Makefile

```diff
@@ -1,9 +1,11 @@
 .PHONY: quality requirements
 
+CHECK_DIRS?=example sortedm2m sortedm2m_tests test_project *.py
+
 quality: ## Run isort, pycodestyle, and Pylint
-	isort --check-only --recursive .
-	pycodestyle example sortedm2m sortedm2m_tests test_project *.py
-	pylint --rcfile=pylintrc example sortedm2m sortedm2m_tests test_project *.py
+	isort --check-only $(CHECK_DIRS)
+	pycodestyle $(CHECK_DIRS)
+	DJANGO_SETTINGS_MODULE=test_project.settings pylint --errors-only --load-plugins pylint_django $(CHECK_DIRS)
```

Comment:
Removing the `--rcfile` is the most opinionated change here - the problem is that I don't have instructions to re-generate the file and the current file is too old and has tons of options that doesn't work anymore. While I can do `pylint --generate-rcfile`, this results in a completely different file from what's there, the diff is so large that it's unhelpful. If I had a set of changes that were previously applied to the file last time it was generated, I could go about re-applying those, but I don't know how to get that.

Because of this, I turned it on in the less-strict mode of `--errors-only`.